src/bp/ui-studio/src/web/translations/en.json

```diff
@@ -0,0 +1,47 @@
+{
+  "general": "General",
+  "status": "Status",
+  "description": "Description",
+  "website": "Website",
+  "phoneNumber": "Phone Number",
+  "contactEmail": "Contact E-mail",
+  "linkToTerms": "Link to Terms & Conditions",
+  "linkToPolicy": "Link to Privacy Policy",
+  "chooseFile": "Choose file",
+  "coverPicture": "Cover Picture",
+  "botAvatar": "Bot Avatar",
+  "saveChanges": "Save Changes",
+  "defaultLanguage": "Default language",
+  "supportedLanguages": "Supported languages",
+  "language": "Language",
+  "name": "Name",
+  "botConfiguration": "Bot Configuration",
+  "additionalDetails": "Additionails Details",
+  "avatarAndCover": "Avatar & Cover picture",
+  "confirmChangeLanguage": "Are you sure you want to change the language of your bot from {currentName} to {newName}? All of your content elements will be copied, make sure you translate them.",
+  "confirmUnmount": "Are you sure want to unmount this bot? All of the functionalities of this bot will become unavailable.",
+  "configUpdated": "Bot configuration updated successfully",
+  "formContainsErrors": "The form contains errors",
+  "requireImageFile": "{targetProp} requires an image file",
+  "details": "Details",
+  "pictures": "Pictures",
+  "avatarUploadSuccess": "The bot avatar has been uploaded successfully. You need to save the form in order for the changes to take effect.",
+  "coverUploadSuccess": "The cover picture has been uploaded successfully. You need to save the form in order for the changes to take effect.",
```

Comment:
I think we shuld group messages specific to a certain section under a namespace. It would keep the translations file more readable in the future.

Ex: `config.coverUploadSuccess`,  `lang.arabic`, `config.status.public`